SampleConsoleClient/SampleConsoleClient.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\UACloudLibraryGraphQLClient\UACloudLibClientLibrary\UACloudLibClientLibrary.csproj" />
+    <ProjectReference Include="..\UACloudLibraryServer\UA-CloudLibrary.csproj" />

```

Comment:
Please remove. The client should be independent of the server.

Reply:
OK